ipyleaflet/leaflet.py

```diff
@@ -978,6 +978,16 @@ def remove_legend_element(self, key):
         self.send_state()
 
 
+class SearchControl(Control):
+    _view_name = Unicode('LeafletSearchControlView').tag(sync=True)
+    _model_name = Unicode('LeafletSearchControlModel').tag(sync=True)
+    url = Unicode().tag(sync=True, o=True)
+    zoom = Int(10).tag(sync=True, o=True)
+    propertyName = Unicode('display_name').tag(sync=True, o=True)
+    propertyLoc = List(['lat', 'lon']).tag(sync=True, o=True)
+    jsonpParam = Unicode('json_callback').tag(sync=True, o=True)
```

Comment:
Attribute names should be in `snake_case` Python side, we have some logic to turn them automatically to `camelCase` on the JavaScript side.

Reply:
Done.